src/components/side_panel/chart/line_bar_pie_panel/design_panel.ts

```diff
@@ -44,9 +48,9 @@ export class LineBarPieDesignPanel extends Component<Props, SpreadsheetChildEnv>
     });
   }
 
-  updateTitle(ev) {
+  updateTitle() {
     this.props.updateChart(this.props.figureId, {
-      title: ev.target.value,
+      title: _t(this.state.title),
```

Comment:
The translation should be at the setup, not here. 

The title of the chart might be translated (we extract chart titles for translation in dashboard), so translatting it at startup make sense. But the new title that the user just inputted will not be in the translation and even if it was a translatable string, we want to store the non-translated one in DB.